client/my-sites/sidebar/sidebar.jsx

```diff
@@ -172,7 +175,7 @@ export const MySitesSidebar = React.createClass( {
 			jetpackEnabled = config.isEnabled( 'manage/themes-jetpack' ),
 			themesLink;
 
-		if ( site && ! site.isCustomizable() ) {
+		if ( site && ! ( site.capabilities && site.capabilities.edit_theme_options ) ) {
```

Comment:
Oh hmm, it looks like we didn't port `isCustomizable` into `getSite` but we can also check the current user capabilities here:
```
import { canCurrentUser } from 'state/current-user/selectors';
//...
const userCanCustomizeSite = canCurrentUser( state, siteId, 'edit_theme_options' );
```